src/main/java/frc/robot/Constants.java

```diff
@@ -162,7 +168,7 @@ public static final class VisionConstants {
                                 Rotation2d.fromDegrees(70),
                                 new Transform3d(
                                         new Translation3d(-0.327, 0.281, 0.333),
-                                        new Rotation3d(0.0, -1.162, 3.14))),
+                                        new Rotation3d(0.0, -0.409, 3.14))),
```

Comment:
do these new values work in sim?

Reply:
They do. I realized I was using the wrong CAD value so the camera would look way up in the air